worlds/tunic/__init__.py

```diff
@@ -341,14 +410,14 @@ def create_regions(self) -> None:
             victory_region.locations.append(victory_location)
 
     def set_rules(self) -> None:
-        if self.options.entrance_rando or self.options.shuffle_ladders:
+        if self.options.entrance_rando or self.options.shuffle_ladders or self.options.grass_randomizer:
             set_er_location_rules(self)
         else:
             set_region_rules(self)
             set_location_rules(self)
 
     def get_filler_item_name(self) -> str:
-        return self.random.choice(filler_items)
+        return self.random.choice([item for item in filler_items if item != "Grass"])
```

Comment:
Is `filler_items` used anywhere else? Why not just remove grass from the list itself?

Reply:
this was mostly me just overthinking, I couldn't decide if it was better to remove it here or from `filler_items` directly, in case we ever needed a list with all possible filler item names for some reason

Reply:
but it probably would be better to just remove it from the list itself